app/models/task.rb

```diff
@@ -1,5 +1,28 @@
 class Task < ApplicationRecord
 
+  def self.populate_from_airtable(user_email)
+    client = Airtable::Client.new(ENV['airtable_api_key'])
+    table = client.table(Setting.last.airtable_base_id, Setting.last.tasks_table_id)
+    records = table.all
+    #Task.where(priority: 2).delete_all unless Task.last.added.today?
+    records.each do |record|
+      shift_assignment_array = record['e_mail:_(from_members)_(from_shift_assignment)']
+      tmpassigneduserstring = ''
+      unless shift_assignment_array.empty?
+        shift_assignment_array.each do |user|
+          tmpassigneduserstring += user
```

Comment:
this variable name tells me nothing about what this variable is for

Reply:
renamed to concated_email_string to indicate that it is a combined string of airtable emails assigned to a certain task